common/views/components/ContentPage/ContentPage.tsx

```diff
@@ -1,45 +1,47 @@
-// @flow
-import { Children, Fragment, createContext } from 'react';
+import {
+  Children,
+  Fragment,
+  createContext,
+  ReactNode,
+  ReactElement,
+} from 'react';
 import Contributors from '../Contributors/Contributors';
-// $FlowFixMe (tsx)
 import Layout8 from '../Layout8/Layout8';
-// $FlowFixMe (tsx)
 import Layout12 from '../Layout12/Layout12';
 import SeriesNavigation from '../SeriesNavigation/SeriesNavigation';
 import PageHeader from '../PageHeader/PageHeader';
 import Outro from '../Outro/Outro';
 import { classNames } from '../../../utils/classnames';
-import type { Node, Element, ElementProps } from 'react';
-// $FlowFixMe (tsx)
-import Body from '../Body/Body';
 import SpacingSection from '../SpacingSection/SpacingSection';
 import SpacingComponent from '../SpacingComponent/SpacingComponent';
-// $FlowFixMe(tsx)
 import CompactCard from '../CompactCard/CompactCard';
 import Image from '../Image/Image';
-// $FlowFixMe (tsx)
 import Space from '../styled/Space';
-// $FlowFixMe (tsx)
 import { WeAreGoodToGo } from '@weco/common/views/components/CovidIcons/CovidIcons';
 import { prismicPageIds } from '@weco/common/services/prismic/hardcoded-id';
-// $FlowFixMe (tsx)
 import BannerCard from '../BannerCard/BannerCard';
+import Body from '../Body/Body';
+import { Season } from '../../../model/seasons';
 /*eslint-disable */
 export const PageBackgroundContext = createContext<'cream' | 'white'>('white');
 
 // TODO: use Element<typeof Component>
-type Props = {|
-  id: string,
-  isCreamy?: boolean,
-  Header: Element<typeof PageHeader>,
-  Body: Element<typeof Body>,
+type Props = {
+  id: string;
+  isCreamy?: boolean;
+  Header: ReactElement<typeof PageHeader>;
+  Body: {
+    props: {
```

Comment:
This seems really un-standard, could you explain why we have to do this differently to everything else?

Reply:
Yes the Element is now not available from react in typescript so i had to change this. 
Function ShouldRenderBody() within the ContentPage,  looks for Body.props.body which does not exists , hence why i put this place. Any other suggestions?

Reply:
@ishouty I've added [this](https://github.com/wellcomecollection/wellcomecollection.org/pull/6046/commits/4b420c696013df5521c06026e657f45cf16a7774). I imagine as we change over from flow, we'll need this a little more.

The problem was [this was the way to do this in flow](https://flow.org/en/docs/react/types/#toc-react-element).